documentation/docs/02-layouts.md

```diff
@@ -76,10 +76,25 @@ For example, if `src/routes/settings/notifications/index.svelte` failed to load,
 
 > SvelteKit provides a default error page in case you don't supply `src/routes/__error.svelte`, but it's recommended that you bring your own.
 
+```ts
+// declaration type
+// * also see type for `LoadOutput` in the Loading section
+
+export interface ErrorLoadInput<
+	PageParams extends Rec<string> = Rec<string>,
+	Stuff extends Rec = Rec,
```

Comment:
I think we should expand `Rec` similarly to the `LoadInput` example in [loading](https://kit.svelte.dev/docs#loading) as it's an internal utility type.

```suggestion
	PageParams extends Record<string, string> = Record<string, string>,
	Stuff extends Record<string, any> = Record<string, any>,
```